### ISSUE TITLE
A11y_AzureDataStudio_BigDataClusterOverviewPage_ScreenReader : VoiceOver is not announcing control type for the buttons such as "Big Data Cluster Overview"

### ISSUE THREAD

**Issue by 12-shweta** [microsoft/azuredatastudio#7813]
"[Check out Accessibility Insights!](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier."

GitHubTags:#A11y_AzureDataStudio;#A11yMAS;#A11yTCS;#WCAG1.3.1;#WCAG4.1.2;#DesktopApp ;#VoiceOver;#SQL Azure Data Studio;#Mac

**Environment Details:**
Application Name: Azure Data Studio
Version: 1.12.0-insider (user setup)
Commit: 6582debd7388b1be512fc3a9b956d3e54f6115e7
Date: 2019-10-03T08:56:27.454Z
VS Code: 1.38.0
Electron: 4.2.10
Chrome: 69.0.3497.128
Node.js: 10.11.0
V8: 6.9.427.31-electron.0
OS:macOS Mojave Version 10.14.6(18G87)

**Additional Details:**
Tool Name : VoiceOver

**Steps to Reproduce:**
1. Install and Launch the application the azure data studio application.
2. Open the “Connections” viewlet (CTRL + SHIFT + D)
3. Navigate to the “SQL Server Big Data Clusters” section and expand it
4. Navigate to the "Add SQL Server big data cluster controller…" node and click on it.
5. Fill in the connection information for the BDC Controller Endpoint
6. Click “Add”
7. Navigate to the controller node in the “SQL Server Big Data Clusters” view and right click on it, then select “Manage” to open the dashboard
8. Start Voiceover.
9. Navigate till "troubleshoot " button  using tab and then use ctrl-option-arrows navigation.
10. Observe the announcement by screen reader when screen reader focus moves to the "Big Data Cluster Overview"

**Actual:**
Screen reader only announce the "Big Data Cluster Overview" and not the control type  for it .

**Expected:**
Screen reader should announce the "Big Data Cluster Overview" and the control type  for it  that is button.

**User Impact:**
Screen reader user will  find it difficult to navigate the screen.

**Attachment for Reference:**
![7813_A11y_AzureDataStudio_BigDataClusterOverviewPage_ScreenReader](https://user-images.githubusercontent.com/54656883/67076205-51c35180-f1aa-11e9-84f1-3e4e8f302031.png)

Does this issue occur when all extensions are disabled?: Yes

**Comment by 12-shweta** [comment]
Verified the bug in below environment :
Version: 1.15.0-insider (user setup)
Commit: 041f34b
Date: 2020-01-08T05:36:19.161Z
VS Code: 1.38.0
Electron: 6.1.6
Chrome: 76.0.3809.146
Node.js: 12.4.0
V8: 7.6.303.31-electron.0
OS: MacOs version 10.15.2 (19C57)
Issue is fixed and does not reproduce .

![image](https://user-images.githubusercontent.com/54656883/72730411-5afff800-3bb7-11ea-9f19-e92cec3c0ee2.png)

**Comment by fsteffi** [comment]
#Azure Data Studio - New Features_Mac;

**Comment by Rama8345** [comment]
A11ySev2

**Comment by Samhitha66** [comment]
#Closed; #Regressed:07-14-23;
